### PR TITLE
Knowteq Jumpsuits

### PR DESCRIPTION
**Pull request by daPotato**
Adds Knowteq Jumpsuits for the Knowteq employees. I refuse to guarantee the functionality of the code.

Please be careful, it is my first time or some shit like that.

### CODE REVIEW

File: code/modules/clothing/under/knowteq.dm [hunk 1]
```diff
@@ -0,0 +1,9 @@
+//Uniforms for Knowteq
+
+/obj/item/clothing/under/knowteq
+	desc = "A blue and gray Jumpsuit worn by Knowteq employees."
```
Comment (ingles98) [inline code/modules/clothing/under/knowteq.dm]: Great! As a KnowTek myself, please make it so desc and name have "KnowTek" instead of "Knowteq"

Comment (HbsRed) [reply code/modules/clothing/under/knowteq.dm]: KnowTeq

Comment (ingles98) [reply code/modules/clothing/under/knowteq.dm]: ;_;

Comment (daPotato) [reply code/modules/clothing/under/knowteq.dm]: screen shot 2018-08-31 at 11 14 26 am

Its right as far as I'm concerned.

Comment (ingles98) [reply code/modules/clothing/under/knowteq.dm]: Alright so it should probably be moved to inside a decently named folder. Not sure if on something like root/custom_stuff/custom_clothing.dm or inside actualy clothing folder. 

Eitherway Faurns probably has an idea for that, probably also implemented, to have hardcoded custom stuff to companies.. Not sure tho

File: code/modules/clothing/under/knowteq.dm [hunk 2]
```diff
@@ -0,0 +1,9 @@
...
+	name = "Knowteq Jumpsuit"
+	icon_state = "knowteq_s"
+	item_state = "knowteq"
+	worn_state = "knowteq_s"
+armor = list(melee = 0, bullet = 0, laser = 0,energy = 0, bomb = 0, bio = 0, rad = 0)
```
Comment (NataKilar) [inline code/modules/clothing/under/knowteq.dm]: Missing a tab here before armor, needs to be fixed before merging

Comment (daPotato) [reply code/modules/clothing/under/knowteq.dm]: Tab was added with the new folder

File: code/modules/clothing/company_clothing/knowteq.dmi [hunk 3]
```diff
@@ -0,0 +1,9 @@
+//Uniforms for Knowteq
+
+/obj/item/clothing/under/knowteq
+	desc = "A blue and gray Jumpsuit worn by Knowteq employees."
+	name = "Knowteq Jumpsuit"
+	icon_state = "knowteq_s"
+	item_state = "knowteq"
+	worn_state = "knowteq_s"
+  armor = list(melee = 0, bullet = 0, laser = 0,energy = 0, bomb = 0, bio = 0, rad = 0)
```
Comment (NataKilar) [inline code/modules/clothing/company_clothing/knowteq.dmi]: Move company_clothing to under, or add a under folder to company_clothing. Furthermore, this file needs to be a .dm, not a .dmi file. Finally, everything from "desc" to "worn_state" needs to have one tab removed.

Comment (daPotato) [reply code/modules/clothing/company_clothing/knowteq.dmi]: oopsie doopsies, wasnt paying attention when i was typing, ill have them fixed shortly

Comment (daPotato) [reply code/modules/clothing/company_clothing/knowteq.dmi]: did i ficks it

File: code/modules/clothing/company_clothing/under.dm [hunk 4]
```diff
@@ -1,11 +1,12 @@
...
-//Knowteq
+//KnowTeq
+
+/obj/item/clothing/company_clothing/under/knowteq
+	desc = "A blue and gray jumpsuit worn by KnowTeq employees."
+	name = "KnowTeq Jumpsuit"
+	icon_state = "knowteq_s"
+	item_state = "knowteq"
+	worn_state = "knowteq_s"
+	armor = list(melee = 0, bullet = 0, laser = 0,energy = 0, bomb = 0, bio = 0, rad = 0)
```
Comment (Kurgis) [inline code/modules/clothing/company_clothing/under.dm]: I don't see any reason to state the armor value, you could just remove it.

**Comment by ingles98** [comment]
Deprecated company - closing.

**Pull request closed without merging**